projects/aws/upgrader/cmd/upgrade_node.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/aws/eks-anywhere-build-tooling/projects/aws/upgrader/upgrade"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var upgradeNodeCmd = &cobra.Command{
+	Use:   "node",
```

Comment:
This naming doesn't seem to follow the pattern of other commands. Kubeadm is not included in the name, only in the description.